### DISCUSSION TITLE
Welcome to js-libp2p Discussions!

### DISCUSSION THREAD

**Discussion by p-shahi** [libp2p/js-libp2p#2025] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tim-mune** [comment]
i am timothy munene a full-stack developer im looking to learn about integrating libp2p into my p2p applications
